.travis.yml

```diff
@@ -1,8 +1,71 @@
 language: node_js
+
+os:
+  - windows
+  - osx
+
+osx_image:
+  # macOS 10.15.5
+  - xcode12
+  # macOS 10.14.6
+  - xcode11.3
+  
 node_js:
-  - '14'
-  - '13'
-  - '12'
-  - '10'
+  - 14
+  - 13
+  - 12
+  - 10
```

Comment:
These should be quotes. Numbers are funky in YAML and it's better to be safe and use strings, as they're actually strings.